src/main/java/org/gridsuite/modification/server/modifications/EquipmentDeletion.java

```diff
@@ -34,7 +34,7 @@ public EquipmentDeletion(EquipmentDeletionInfos modificationInfos) {
 
     @Override
     public void apply(Network network, Reporter subReporter) {
-        Identifiable<?> identifiable = ModificationUtils.getInstance().getEquipmentByIdentifiableType(network, modificationInfos.getEquipmentType(), modificationInfos.getEquipmentId());
+        Identifiable<?> identifiable = ModificationUtils.getInstance().getEquipmentByIdentifiableType(network, modificationInfos.getEquipmentType().name(), modificationInfos.getEquipmentId());
```

Comment:
Double conversion ?

Reply:
@SlimaneAmar I updated the code to avoid the double conversion. I note that regulating terminal type also use a String https://github.com/gridsuite/network-modification-server/blob/c535de666d98c178be5051be8ff6c00a06301870/src/main/java/org/gridsuite/modification/server/dto/TapChangerCreationInfos.java#L45